src/test/java/io/openliberty/tools/intellij/it/SingleModMPProjectTestCommon.java

```diff
@@ -1007,6 +1020,67 @@ public static void deleteDirectoryIfExists(String dirPath) {
         }
     }
 
+    /**
+     * Clean project.
+     */
+    public void stopTerminal() {
+        if (!shouldCleanupTerminal) {
+            return;
+        }
+
+        Keyboard keyboard = new Keyboard(remoteRobot);
+        ProjectFrameFixture projectFrame = remoteRobot.find(ProjectFrameFixture.class, Duration.ofSeconds(10));
+        ComponentFixture terminal = remoteRobot.find(ComponentFixture.class, byXpath("//div[@class='JBTerminalPanel']"), Duration.ofSeconds(10));
+
+        terminal.rightClick();
+        ComponentFixture openFixtureNewTab = projectFrame.getActionMenuItem("New Tab");
+        openFixtureNewTab.click(new Point());
+
+        // Perform clean
+        String projectName = getSmMPProjectName();
+        if ("singleModMavenMP".equalsIgnoreCase(projectName)) {
+            keyboard.enterText("mvn liberty:stop");
+
+        } else if ("singleModGradleMP".equalsIgnoreCase(projectName) || "singleMod GradleMP".equalsIgnoreCase(projectName)) {
+            keyboard.enterText("gradle libertyStop");
+        }
+        keyboard.enter();
+        TestUtils.sleepAndIgnoreException(5);
+    }
+
+    /**
+     * Stop the Server.
+     */
+    public void cleanTerminal() {
+        if (!shouldCleanupTerminal) {
+            return;
+        }
+
+        Keyboard keyboard = new Keyboard(remoteRobot);
+        // Perform clean
+        String projectName = getSmMPProjectName();
+        if ("singleModMavenMP".equalsIgnoreCase(projectName)) {
+            keyboard.enterText("mvn clean");
```

Comment:
Actually this applies above as well: these test cases use the maven and gradle wrappers. It would be cleaner if you stayed consistent with this and used the wrappers here (`mvnw` or `gradlew` or the Windows version). Some people may run this on local machines and they may not have maven or gradle installed locally or the local version may be an older version.

Reply:
Addressed the comment and made changes.